README.md

```diff
@@ -286,9 +292,18 @@ ua = 'ckanapiexample/1.0 (+http://example.com/my/website)'
 
 demo = RemoteCKAN('https://demo.ckan.org', user_agent=ua)
 showcases= demo.action.ckanext_showcase_list()
-print showcases
+print(showcases)
 ```
 
+Combining query parameters clauses is possible as in the following `package_search` action.  This query combines three clauses that are all satisfied by the single [example dataset](https://demo.ckan.org/dataset/sample-dataset-1) in the Demo CKAN site. 
+
+More detailed complex query syntax examples can be found in the [SOLR documentation](https://solr.apache.org/guide/6_6/common-query-parameters.html).
+
+```python
+packages = demo.action.package_search(q='+organization:sample-organization +res_format:GeoJSON +tags:geojson')
+print(packages)
+```  
+
```

Comment:
this seems more suitable as a script in the examples directory (this is information about using a specific ckan endpoint, not about how to use the ckanapi library). The descriptive text might be a good thing to add to the ckan `package_search` docstring itself.

Reply:
Actually, the intent was more to provide an example of how to combine clauses together for a single query parameter (space between each).  It applies to any CKAN.  When I was reading the existing README examples, I couldn't find another one that had any sort of complex SOLR query shown.  Someone can always go back to the CKAN API or SOLR docs themselves, but a readily-available example is helpful too.

Reply:
I thought I had committed the RemoteCKAN lines of this code block before issuing the PR but I guess I hadn't, maybe this helps illustrate the example and keeps things more consistent with existing examples.